src/linking/static_lifetime_init.cpp

```diff
@@ -127,14 +127,25 @@ void static_lifetime_init(
     symbols.insert(id2string(symbol_pair.first));
   }
 
+  // first do framework variables
   for(const std::string &id : symbols)
-  {
-    auto code = static_lifetime_init(id, symbol_table);
-    if(code.has_value())
-      dest.add(std::move(*code));
-  }
+    if(has_prefix(id2string(id), CPROVER_PREFIX))
```

Comment:
The use of `id2string` seems unnecessary, we are dealing with a `std::string` already.

Reply:
Fixed